src/ApiClient.php

```diff
@@ -322,6 +322,30 @@ public function getBookVariables($id)
         return $this->handleResult($requestResult);
     }
 
+    /**
+     * Change varible by user email
+     *
+     * @param int $bookID
+     * @param string $email User email
+     * @param array $data User vars in [key=>value] format
+     * @return stdClass
+     */
+    public function updateEmailVariables(int $bookID, string $email, array $data)
+    {
+        if (empty($bookID)) {
+            return $this->handleError('Empty book id');
+        }
+
+        $vars = ['email' => $email, 'variables' => []];
```

Comment:
Use name '$data' instead '$vars'. 
This is just semantic nuances, cause we used to use $data as payload for request